docs/data-modeling-lifecycle/tutorial-1-knowledge-acquisition.md

```diff
@@ -393,12 +393,6 @@ TODO Step by step guide on how to visualize the shared data model.
 ## DMS Architect: Alice
 
 ### Implementing the Enterprise Data Model in CDF
-Neat supports exporting a data model to CDF either using an `AssetLoader` or `DMSLoader`. The `AssetLoader` is
-used to load the data model into the classical `AssetHierarchy` in CDF, while the `DMSLoader` is used to load the
-data model into the new Data Modeling Service (DMS) in CDF. The `DMSLoader` is the recommended way to load the
-data model into CDF, as it supports more advanced features such as defining dependencies between data. In this
-tutorial, we will focus on the `DMSLoader`.
```

Comment:
I think this is just confusing so removing it for clarity.

Reply:
Agree!